torch/csrc/jit/passes/peephole_list_idioms.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <torch/csrc/jit/ir/ir.h>
+
+namespace torch {
+namespace jit {
+
+// Optimizes list idioms. Currently this is invoked as part of PeepholeOptimize
```

Comment:
Can we get a more descriptive comment, maybe a before/after pseudocode? I'm not sure what a list idiom is at first reading